File: VB/DXRichEditControlAPISample/CodeExamples/FormFieldsActions.vb
Imports DevExpress.XtraRichEdit.API.Native
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks

Namespace DXRichEditControlAPISample.CodeExamples
    Friend Class FormFieldsActions
        Private Shared Sub CreateCheckbox(ByVal document As Document)
'            #Region "#CreateCheckbox"
            Dim currentPosition As DocumentPosition = document.CaretPosition
            Dim checkBox As DevExpress.XtraRichEdit.API.Native.CheckBox = document.FormFields.InsertCheckBox(currentPosition)
            checkBox.Name = "check1"
            checkBox.State = CheckBoxState.Checked
            checkBox.SizeMode = CheckBoxSizeMode.Auto
            checkBox.HelpTextType = FormFieldTextType.Custom
            checkBox.HelpText = "help text"
'            #End Region ' #CreateCheckbox
        End Sub
    End Class
End Namespace

File: CS/DXRichEditControlAPISample/CodeExamples/FormFieldsActions.cs
using DevExpress.XtraRichEdit.API.Native;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DXRichEditControlAPISample.CodeExamples
{
    class FormFieldsActions
    {
        static void CreateCheckbox(Document document)
        {
            #region #CreateCheckbox
            DocumentPosition currentPosition = document.CaretPosition;
            DevExpress.XtraRichEdit.API.Native.CheckBox checkBox = document.FormFields.InsertCheckBox(currentPosition);
            checkBox.Name = "check1";
            checkBox.State = CheckBoxState.Checked;
            checkBox.SizeMode = CheckBoxSizeMode.Auto;
            checkBox.HelpTextType = FormFieldTextType.Custom;
            checkBox.HelpText = "help text";
            #endregion #CreateCheckbox
        }
    }
}
